plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -306,6 +374,9 @@ fu_ccgx_hpi_device_reg_read (FuCcgxHpiDevice *self,
 		g_prefix_error (error, "read error: ");
 		return FALSE;
 	}
+
+	/* 5 ms */
+	g_usleep (5000);
```

Comment:
Why is this one different than `HPI_CMD_FLASH_READ_WRITE_DELAY_US`

Reply:
Yes, it's all a bit muddled. I've moved them up into the read/write functions to be super explicit.